scripts/zones/Uleguerand_Range/mobs/Mountain_Worm_NM.lua

```diff
@@ -0,0 +1,24 @@
+-----------------------------------
+-- Area: Uleguerand Range
+--  Mob: Mountain Worm NM
+--  https://www.bg-wiki.com/ffxi/Mountain_Worm_(NM)
+-- TODO allow deaggro based on distance (core CMobEntity::CanDeaggro() forces NM and Battlefield mobs to never stop chasing)
+-----------------------------------
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMagicCastingEnabled(false)
```

Comment:
missed a spot when removing the onMobFight part

Reply:
Oops, thank you

Reply:
updated